python/lsst/ts/IntegrationTests/config_registry.py

```diff
@@ -0,0 +1,53 @@
+# This file is part of ts_IntegrationTests
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+from collections import UserDict
+
+__all__ = ["registry"]
+
+
+class ConfigRegistry(UserDict):
+    """Add the given named configuration to the registry dictionary.
```

Comment:
The documentation is in the wrong place. This should be in the `__setitem__` function not at the class level.

Reply:
Rearranged as requested.